tools/run_tests/xds_k8s_test_driver/tests/authz_test.py

```diff
@@ -107,19 +107,19 @@ def authz_rules(self):
                     },
                 },
             },
-            {
-                "sources": {
-                    "principals": ["*"],
-                },
-                "destinations": {
-                    "hosts": [f"*:{self.server_xds_port}"],
-                    "ports": [self.server_port],
-                    "httpHeaderMatch": {
-                        "headerName": "test",
-                        "regexMatch": "principal-present",
-                    },
-                },
-            },
+            # {
```

Comment:
Add a comment about b/202058316? So, we won't forget?

Reply:
Well, none of the below can be uncommented without this being uncommented. But went ahead and added a comment here with a brief description.